src/extensions.rs

```diff
@@ -85,19 +92,31 @@ impl CommandExt for Command {
     }
 
     fn fmt_message(&self, mut msg_info: &mut MessageInfo) -> String {
+        use std::fmt::Write;
         let msg_info = &mut msg_info;
+        let mut string = String::new();
+        if let Some(caller) = msg_info.caller() {
+            write!(string, "[{}] + ", caller).unwrap();
+        } else {
+            write!(string, "+ ",).unwrap();
```

Comment:
trailing ,

Reply:
Is there supposed to be a trailing comma? The previous `write!` doesn't have it, but this one does.